### DISCUSSION TITLE
Custom Keymap to exit?

### DISCUSSION THREAD

**Discussion by TLSingh1** [mikavilpas/yazi.nvim#274] [General]
Hi, thank you so much for your work! my new favorite file tree BY FAR!!!

Question regarding overriding yazi keymaps:
`    set_keymappings_function = function(yazi_buffer_id, config, context) end,`

Forgive me, but how would I remap ; to close yazi.nvim?

**Comment by mikavilpas** [comment]
Awesome! 

I was thinking it might be the easiest to map a key in yazi itself. That way you would have the same key close yazi as well as yazi.nvim (yazi.nvim simply closes when yazi exits).

Would this work for you?

**Reply by TLSingh1** [reply]
Perfect, thanks so much!
